selfie.c

```diff
@@ -2349,6 +2353,10 @@ void put_character(uint64_t c) {
 
     output_cursor = output_cursor + 1;
   } else {
+    if (!init_done) {
```

Comment:
@maroneze The `!` operator is not supported in C*. Could you please replace it with `init_done == 0`? Thanks!

Reply:
Also, please avoid using `{` and `}` if there is only one statement in between. Thanks!